config/application.rb

```diff
@@ -141,5 +142,8 @@ class Application < Rails::Application
     config.active_storage.queues.preview_image = :active_storage
     config.active_storage.queues.purge = :active_storage
     config.active_storage.queues.transform = :active_storage
+
+    # Use secret from archive config
+    config.secret_key_base = ArchiveConfig.SESSION_SECRET
```

Comment:
According to the Rails documentation, it's preferred to do this in `credentials.yml.enc` instead: https://guides.rubyonrails.org/v7.1/configuring.html#config-secret-key-base / https://api.rubyonrails.org/v7.1.5.1/classes/Rails/Application.html#method-i-secret_key_base